ceaser.py

```diff
@@ -0,0 +1,30 @@
+"""This script prompts a user to enter a message to encode or decode
+using a classic Caeser shift substitution"""
+
+import string
+
+def encode(message, offset):
+    """ Encode function """
+    encoded = ''
+    letters = string.ascii_letters + string.punctuation + string.digits
+    for letter in message:
+        if letter == ' ':
+            encoded = encoded + ' '
+        else:
+            pos = letters.index(letter) + offset
+            encoded = encoded + letters[pos]
+
+    return encoded
+
+def decode(encoded_message, offset):
+    """ Decode function """
+    encoded = ''
+    letters = string.ascii_letters + string.punctuation + string.digits
+    for letter in encoded_message:
+        if letter == ' ':
```

Comment:
The same thing is happening here. Add another test for decode() that allows for this condition to also be true so that line 25 gets coverage.